service/job/jobs.py

```diff
@@ -292,26 +298,32 @@ def _create_chains(self, params, user_name):
             current_chain |= _link('generate_xml',
                                    template_file='ojs3_template_issue.xml',
                                    target_filename='ojs_import.xml',
-                                   ojs_metadata=params['options']['ojs_metadata'])
+                                   ojs_options=params['options']['ojs_options'])
 
             current_chain |= _link('generate_xml',
                                    template_file='mets_template_no_articles.xml',
                                    target_filename='mets.xml',
                                    schema_file='mets.xsd')
+
             if params['options']['ocr_options']['do_ocr']:
                 current_chain |= _link('list_files',
                                        representation='tif',
                                        target='txt',
                                        task='convert.tif_to_txt',
                                        ocr_lang=params['options']['ocr_options']['ocr_lang'])
 
-            if params['options']['ojs_metadata']['auto_publish_issue']:
+            if params['options']['ojs_options']['auto_publish_issue']:
                 current_chain |= _link('publish_to_ojs',
-                                       ojs_metadata=params['options']['ojs_metadata'],
+                                       ojs_options=params['options']['ojs_options'],
                                        ojs_journal_code=issue_target['metadata']['ojs_journal_code'])
+
             current_chain |= _link('publish_to_repository')
             current_chain |= _link('publish_to_archive')
-            current_chain |= _link('cleanup_workdir')
+            current_chain |= _link('cleanup_directories',
+                                    keep_staging=params['options']['app_options']['keep_staging'], 
+                                    staging_current_folder=issue_target['path'],
+                                    user_name=user_name)
+                                    
```

Comment:
Ich fände es schlanker/einheitlicher wenn:

a) Getestet wird, ob keep_staging auf true ist, um nur dann den Task in die Chain aufzunehmen (analog zu den ojs options).
b) Anschließend nur noch folder und user übergeben.